src/main/python/SentimentAnalyzer.py

```diff
@@ -0,0 +1,47 @@
+import pandas as pd
+# pip install --upgrade azure-ai-textanalytics
+from azure.ai.textanalytics import TextAnalyticsClient
+from azure.core.credentials import AzureKeyCredential
+
+_key = "adc96f39ac2a4918a1f462b5bf8674b8"
+_endpoint = "https://nbatwittersentiment.cognitiveservices.azure.com/"
+
+def _authenticate(key:str,endpoint:str):
+    # Authentication
+    ta_credential = AzureKeyCredential(key)
+    client = TextAnalyticsClient(
+        endpoint=endpoint,
+        credential=ta_credential,
+        api_version='TextAnalyticsApiVersion.V3_0')
+    return client
+
+def _formatresponse(responseDict) -> pd.DataFrame :
+    col_list = responseDict.keys()
+    cleanresponse = {'tweetID':responseDict.id,
+                    'positive':responseDict.confidence_scores.positive,
+                    'neutral':responseDict.confidence_scores.neutral,
+                    'negative':responseDict.confidence_scores.negative
+                    }
+    for idx, sentence in enumerate(responseDict.sentences):
+        cleanresponse ['tweet']= sentence.text
+    # sentiment_df = pd.DataFrame.from_dict(cleanresponse,orient='index')
+    sentiment_df = pd.DataFrame.from_records(cleanresponse,index=[0])
```

Comment:
What does this do?

Reply:
removed the commented line. I was not sure what orientation we needed, I matched the output to the other sentiment analyzer from @el_Grudge. Since this client can take in multiple or single documents, I needed to step through the list to pull the text. This should work for one or multiple tweets submitted. I also pulled the overall scores for the request. If more than one tweet is submitted the overall sentiment is returned.

Reply:
I more meant what the from_records does lol. I can just look that up though. Figured I'd ask in case you had a quick thing you could say.